products/compute/api.yaml

```diff
@@ -12745,6 +12755,12 @@ objects:
           creation/deletion.
         output: true
       # 'storageBytesStatus' not useful for object convergence.
+      - !ruby/object:Api::Type::Array
+        name: 'storageLocations'
+        description: |
+          Cloud Storage bucket storage location of the snapshot (regional or multi-regional).
+        item_type: Api::Type::String
+        output: true
```

Comment:
Should this be `output:true`? It looks to be configurable.

Reply:
fixed